docker-compose.yml

```diff
@@ -0,0 +1,112 @@
+version: '3.7'
+services:
+    mysql:
+        image: mariadb
+        ports:
+            - '3306'
+        environment:
+            MYSQL_ROOT_PASSWORD: password
+            MYSQL_DATABASE: wordpress
+        volumes:
+            - 'mysql:/var/lib/mysql'
+    tests-mysql:
+        image: mariadb
+        ports:
+            - '3306'
+        environment:
+            MYSQL_ROOT_PASSWORD: password
+            MYSQL_DATABASE: tests-wordpress
+        volumes:
+            - 'mysql-test:/var/lib/mysql'
+    wordpress:
+        build: .
+        depends_on:
+            - mysql
+        image: wordpress
+        ports:
+            - '${WP_ENV_PORT:-8888}:80'
+        environment:
+            WORDPRESS_DB_USER: root
+            WORDPRESS_DB_PASSWORD: password
+            WORDPRESS_DB_NAME: wordpress
+        volumes: &ref_0
+            - >-
+                /Users/ockham/.wp-env/f225b6dcecc0956ba67033945142a55c/WordPress:/var/www/html
+            - >-
+                /Users/ockham/src/gutenberg:/var/www/html/wp-content/plugins/gutenberg
+            - >-
+                /Users/ockham/.wp-env/f225b6dcecc0956ba67033945142a55c/theme-experiments/tt1-blocks:/var/www/html/wp-content/themes/tt1-blocks
```

Comment:
A supplementary ("3rd-party") theme, needed for developing the primary plugin. Defined by the latter.

Relevant concern: Dependency management.